internal/conf/validate_custom.go

```diff
@@ -62,6 +62,11 @@ func validateCustom(cfg Unified) (problems Problems) {
 		}
 	}
 
+	// Prevent default licence key from being set
+	if cfg.LicenseKey == "<license key>" {
```

Comment:
wondering if I can get away with:

```suggestion
	if strings.HasPrefix(cfg.LicenseKey, "<") {
```

or similar

Reply:
I assume our license keys conform to some format. If I am not mistaken they are base64 encoded strings. So need to match the regex `[A-Za-z0-9+/]+=*`. I could be wrong about that though and haven't confirmed it.

Reply:
Oh it seems we also support ignoring whitespace. It might be better to just see if we can parse a license key here using this function: https://sourcegraph.com/github.com/sourcegraph/sourcegraph@v3.16.0/-/blob/enterprise/internal/license/license.go?subtree=true#L130

Reply:
I think that's what I'm trying to do here: https://github.com/sourcegraph/sourcegraph/pull/11008/files#diff-7ad727282237639e904229068bc25564R8

Should I remove the check here in favour of that?